packages/admin-ui/ui/src/domain/product-categories/components/product-category-list-item-details.tsx

```diff
@@ -50,6 +49,8 @@ function ProductCategoryListItemDetails(
       onClick: async () => {
         try {
           await deleteCategory()
+          await queryClient.invalidateQueries(adminProductKeys.all)
```

Comment:
can we do this in the `useAdminDeleteProductCategory` hook similar to what we do with `useAdminUpdateProductCategory`?